pkg/apis/pipeline/v1/pipelinerun_types.go

```diff
@@ -469,6 +469,8 @@ const (
 	TasksTimedOutSkip SkippingReason = "PipelineRun Tasks timeout has been reached"
 	// FinallyTimedOutSkip means the task was skipped because the PipelineRun has passed its Timeouts.Finally.
 	FinallyTimedOutSkip SkippingReason = "PipelineRun Finally timeout has been reached"
+	// EmptyArrayInMatrixParams means the task was skipped because Matrix parameters contain empty array.
+	EmptyArrayInMatrixParams SkippingReason = "PipelineRun tasks were skipped because matrix parameters contain empty array"
```

Comment:
```suggestion
	EmptyArrayInMatrixParams SkippingReason = "Matrix Parameters have an empty array"
```

Reply:
the previous reasons reference "pipelinerun tasks / pipelinerun finally" because they are talking about the timeouts set in pipelineruns

Reply:
see `MissingResultsSkip` or `WhenExpressionsSkip` for other examples

Reply:
Ah I see. Thanks!